README.md

```diff
@@ -1 +1,159 @@
-# vet_clinic
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [vet-clinic] <a name="about-project"></a>
+
+In this project I developed the foundational structure for a vet clinic project, including the implementation of a relational database. Within this project, I designed and populated a table to store comprehensive information about animals. Additionally, I enabled the capability to insert data into the table and perform queries on it.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Proper addition of animals table]**
+- **[Proper insertion of animal data into table]**
+- **[Diversification of queries in table]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Git](https://git-scm.com/)
+- [Node](https://nodejs.org/en/)
+- A code editor, such as [VSCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone git@https://github.com/everinurmind/vet_clinic.git
+```
+
+### Install
+
+Install this project dependencies with:
+
+```bash
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+  npm run start
```

Comment:
- [ ] To make your documentation be more professional and descriptive to non technical users, I think it would be better if you update information you added above by using correct and appropriate information which related to your project such as command of how to `access` the cloned directory, `tools needed`(e.g.: Postgress,...) to run your project locally, hoe to `open` and `use` Postgresql,...